### PR TITLE
$comment -> $reason

### PR DESCRIPTION
**Pull request by MacFan4000**
Part of this prevented the reason from being placed in the farmer log as before

### CODE REVIEW

File: SpecialRequestWiki.php [hunk 1]
```diff
@@ -147,7 +147,7 @@ public function onSubmit( array $formData ) {
 		$farmerLogEntry = new ManualLogEntry ( 'farmer', 'requestwiki' );
 		$farmerLogEntry->setPerformer( $this->getUser() );
 		$farmerLogEntry->setTarget( $this->getTitle() );
-		$farmerLogEntry->setComment( $comment );
+		$farmerLogEntry->setComment( $reason );
```
Comment (paladox) [inline SpecialRequestWiki.php]: Should be $formData['reason'],

Comment (MacFan4000) [reply SpecialRequestWiki.php]: @paladox I will mention that $reason is used elsewhere in the file

File: SpecialRequestWiki.php [hunk 2]
```diff
@@ -147,7 +147,7 @@ public function onSubmit( array $formData ) {
 		$farmerLogEntry = new ManualLogEntry ( 'farmer', 'requestwiki' );
 		$farmerLogEntry->setPerformer( $this->getUser() );
 		$farmerLogEntry->setTarget( $this->getTitle() );
-		$farmerLogEntry->setComment( $comment );
+		$farmerLogEntry->setComment( 'reason' );
```
Comment (paladox) [inline SpecialRequestWiki.php]: Should be $formData['reason'],

**Comment by Reception123** [comment]
I am unsure about this, but in any case I think "don't fix what's not broken" applies here

File: SpecialRequestWiki.php (hunk 2 above)
Comment (MacFan4000) [reply SpecialRequestWiki.php]: @JohnFLewis what should it be?

**Comment by Reception123** [comment]
Even though this was removed, the system seems to be working now, so unless we are sure about the correct way to do this, it shouldn't be merged.

File: SpecialRequestWiki.php (hunk 2 above)
Comment (JohnFLewis) [reply SpecialRequestWiki.php]: $formData['reason']

**Comment by JohnFLewis** [comment]
@Reception123 this is fixing something that is broken. It's a change in behaviour that's quite big in the logs. Unless this is the now preferred behaviour (inadvertently) then the whole setComment line should be removed.

**Pull request merged**